manifests/plugin/discovery.pp

```diff
@@ -33,15 +33,19 @@
   if $install_images {
     include ::tftp::params
 
-    foreman::remote_file {"${::tftp::params::root}boot/${kernel}":
-      remote_location => "${source}${kernel}",
+    foreman::remote_file {"${::tftp::params::root}boot/${image_name}":
```

Comment:
I think I don't get the example. Is this what you mean?

```diff
diff --git a/manifests/plugin/discovery.pp b/manifests/plugin/discovery.pp
index 334f63e..6191dbc 100644
--- a/manifests/plugin/discovery.pp
+++ b/manifests/plugin/discovery.pp
@@ -23,6 +23,7 @@ class foreman::plugin::discovery (
   $source         = $foreman::plugin::discovery::params::source,
   $image_name     = $foreman::plugin::discovery::params::image_name,
   $install_images = $foreman::plugin::discovery::params::install_images,
+  $tftp_root      = $tftp::params::root,
 ) inherits foreman::plugin::discovery::params {
```

In the example above it's different.